worker.js

```diff
@@ -113,106 +97,110 @@ const main = module.exports = function () {
      * that we can borrow implementations from.
      */
 
-      const maybePromise = this.lambdaFunction.handler(eventObject, {}, (err, result) => {
-        if (!result) {
-          this.sendError(id, err)
-          return
-        }
+    console.error('this.handler field', this.handler)
```

Comment:
should this be here?

Reply:
This is a dead print i added for debugging, needs to be removed.